MAINTAINERS

```diff
@@ -27,3 +27,4 @@ Lubron Zhan , lubronzhan , VMware
 Xiongxiong Yuan , yxxhero , Qunar
 Marcel Mueller , MarcelMue , Giant Swarm
 Thomas Coudert , thcdrt , OVHcloud
+Nicolas Lacroux , nicolaslacroux , OVHcloud
```

Comment:
You might need `contributor` permissions and so on. No problem for me but you will likely have to talk to @OrlinVasilev